### ISSUE TITLE
Enable Nabla for Pendulum

### ISSUE THREAD

**Issue by TorstenStueber** [pendulum-chain/portal#452]
Enable Nabla for Pendulum and connect to the Pendulum prototype deployment (router address `6buMJsFCbXpHRyacKTjBn3Jss241b2aA7CZf9tKzKHMJWpcJ`).

**Comment by vadaynujra** [comment]
@TorstenStueber maybe something you could hand over to @Sharqiewicz ?

**Comment by TorstenStueber** [comment]
Not really. These changes are trivil (see https://github.com/pendulum-chain/portal/pull/453/files and https://github.com/pendulum-chain/portal/pull/453/files for the two other Nabla deployments).
